.env.sample

```
@@ -2,4 +2,7 @@ CLIENT_ID=1030124019434934412
 GUILD_ID=1037490021462179910
 TOKEN=MTAzMDEyNDAxOTQzNDkzNDQxMg.Gk_rG2.euJJRK-SZCc4X-r1Q_xTuB5TWm937fuAG2Fbb8
 RELAUNCH_DATE=2 #one day
-DATABASE='mongodb+srv://root:root@cluster0.mgy22jx.mongodb.net/test'
\ No newline at end of file
+DATABASE=104.248.137.224:1547
+DB_USER=tcmongo
+DB_PASSWORD=T0g3th3rCr3wM0ng0P55
+LOCAL='mongodb+srv://root:root@cluster0.mgy22jx.mongodb.net/test'
```

Please DO NOT share credentials

```suggestion
DATABASE=XXX
DB_USER=XXX
DB_PASSWORD=XXX
LOCAL=XXX
```